setup.cfg

```diff
@@ -32,9 +32,12 @@ install_requires =
     invenio-administration>=2.0.0,<3.0.0
     invenio-jobs>=1.0.0,<2.0.0
     lxml>=4.5.0
+    # Upper pinning pycountry due to commonmeta-py
+    pycountry>=22.3.5,<23.0.0
     PyYAML>=5.4.1
     regex>=2024.7.24
     rdflib>=7.0.0
+    SPARQLWrapper>=2.0.0
```

Comment:
minor: we make these dependencies extras, since they might be complicating the dependency graph and cause trouble downstream.